mkdocs.yml

```diff
@@ -0,0 +1,45 @@
+# MkDocs configuration for builds with material theme
+
+INHERIT: mkdocs-base.yml
+
+site_url: "https://docs.percona.com/percona-distribution-for-mongodb/"
```

Comment:
I'm not sure but seems we need a version here too

Reply:
Updated